### ISSUE TITLE
409 error

### ISSUE THREAD

**Issue by Bouki6** [MrRoundRobin/telegram.bot#93]
Hello. 
sometimes user block the bot or bot sending rate, hit the limit and cause the 409 error. but in both situations API throw same `System.Net.Http.HttpRequestException`.
```
System.AggregateException: One or more errors occurred. ---> System.Net.Http.HttpRequestException: Response status code does not indicate success: 403 (Forbidden).
   at System.Net.Http.HttpResponseMessage.EnsureSuccessStatusCode()
   at Telegram.Bot.Api.<SendWebRequest>d__78`1.MoveNext()
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at TelegramGateWay.ViaTelegram.SendPhoto()
---> (Inner Exception #0) System.Net.Http.HttpRequestException: Response status code does not indicate success: 403 (Forbidden).
   at System.Net.Http.HttpResponseMessage.EnsureSuccessStatusCode()
   at Telegram.Bot.Api.<SendWebRequest>d__78`1.MoveNext()<---
```
clearly the exception type should be `Telegram.Bot.Types.ApiRequestException` and the exception description be something like this:
[Error]: Bot was blocked by the user. 
i really appreciated if you can fix this or at least give a time frame.
thank you.

**Comment by MrRoundRobin** [comment]
I am currently quite busy with my bachelor thesis, I may have time next week

**Comment by Bouki6** [comment]
> I am currently quite busy with my bachelor thesis, I may have time next week.
Thank you. i look forward to it.

**Comment by MrRoundRobin** [comment]
should be fixed in e72c9207f40807123919f52e3d9175756ba8b30f
